tests/collaborative/collaborative.test.ts

```diff
@@ -997,15 +997,16 @@ describe("Multi users synchronisation", () => {
     const firstSheetId = alice.getters.getActiveSheetId();
     network.concurrent(() => {
       setStyle(bob, "A1", { fontSize: 36 });
+      setCellContent(bob, "A1", "text");
       charlie.dispatch("DUPLICATE_SHEET", {
         sheetId: firstSheetId,
         sheetIdTo: "sheet2",
       });
       charlie.dispatch("DELETE_SHEET", { sheetId: firstSheetId });
     });
-    expect([alice, bob, charlie]).toHaveSynchronizedValue(
+    expect([bob, charlie]).toHaveSynchronizedValue(
```

Comment:
you cannot remove Alice here- the very goal of the tests is to make sure that every user has synchronized values ;-)

Reply:
> you cannot remove Alice here- the very goal of the tests is to make sure that every user has synchronized values ;-)

@rrahir Change done. I thought it was enough to use `alice` to calculate the cell height to show `alice` also have this synchronized value

Reply:
in this very example it is, but it's better to keep a standard way to test the synchronization because makein assumptions could be problematic in the future if we were to change the assertion tested :)